omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -280,31 +257,30 @@ OMERO configuration
 
     $ source ~/.bash_profile
 
-2. To use Omero, we need to first set up Postgres. Open a command-line terminal and run the
+2. To use OMERO, we need to first set up Postgres. Open a command-line terminal and run the
    following commands to create a user called *db_user* and database called *omero_database*::
 
     $ createuser -w -D -R -S db_user
     $ createdb -E UTF8 -O db_user omero_database
 
-3. Create directory for OMERO to store its data::
-
-    $ mkdir -p ~/OMERO
-
-4. Start your Virtualenv environment we created earlier::
-
-    $ source ~/Virtual/omero/bin/activate
+3. Now set the OMERO configuration::
 
-5. Now set the OMERO configuration::
-
-    $ omero config set omero.data.dir ~/OMERO
     $ omero config set omero.db.name omero_database
     $ omero config set omero.db.user db_user
     $ omero config set omero.db.pass db_password
 
-6. Create and run script to initialize the OMERO database::
+4. Create and run script to initialize the OMERO database::
 
     $ omero db script --password omero -f - | psql -h localhost -U db_user omero_database
 
+Binary Repository
+^^^^^^^^^^^^^^^^^
+
+1. Create directory for OMERO to store its data::
+
+    $ mkdir -p ~/OMERO
```

Comment:
I'd suggest another location. On my Mac (where `~/omero` already exists):

```
# joshuas-mbp:~ jamoore$ ls omero
editor       filesOmero   importer.ini log          sessions     tmp
# joshuas-mbp:~ jamoore$ mkdir OMERO
mkdir: OMERO: File exists
```

i.e. we have a capitalization conflict.